plugins/inputs/cloudwatch/cloudwatch.go

```diff
@@ -19,14 +21,16 @@ import (
 
 type (
 	CloudWatch struct {
-		Region      string `toml:"region"`
-		AccessKey   string `toml:"access_key"`
-		SecretKey   string `toml:"secret_key"`
-		RoleARN     string `toml:"role_arn"`
-		Profile     string `toml:"profile"`
-		Filename    string `toml:"shared_credential_file"`
-		Token       string `toml:"token"`
-		EndpointURL string `toml:"endpoint_url"`
+		Region           string   `toml:"region"`
+		AccessKey        string   `toml:"access_key"`
+		SecretKey        string   `toml:"secret_key"`
+		RoleARN          string   `toml:"role_arn"`
+		Profile          string   `toml:"profile"`
+		Filename         string   `toml:"shared_credential_file"`
```

Comment:
Since this a pre-existing plugin, changing options isn't smiled upon. Perhaps for 2.0? Unless it makes sense to add in `CredentialPath` and start deprecating `Filename`. This is slated for a minor release, so I'm fine with some config options improving

Reply:
I guess to users, it's still `shared_credential_file`... I'll update the code to be more clear and use `CredentialPath`

Reply:
It was likely `Filename` because that's what it is in our aws `CredentialConfig`